examples/examples-cpu/nyc-taxi-snowflake/dashboard.ipynb

```diff
@@ -61,13 +61,14 @@
     "import datetime as dt\n",
```

Comment:
<p>this autoformat seems strange to me, I would prefer the old version</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>oy yeah I'd prefer the old way too. But the point of <code>black</code>  is that you give up caring about exactly what it looks like, and what you get is the freedom of never thinking about it again.</p><p> </p><p>I think we can trick this into giving us what we want by adding another header like <code>Accept: "application/json"</code>.  Honestly we should have had that anyway.</p>

Reply:
ok I think this had the desired effect! https://github.com/saturncloud/examples/pull/60/commits/6869031b618eb9c7709864185fab0ac0a6513534

![image](https://user-images.githubusercontent.com/7608904/103698360-ddb7c980-4f66-11eb-9194-b7bfb86b9158.png)